File: Placements.DataAccess/placement/Models/Trainer.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Trainer
{
    public long Id { get; set; }

    public string? Name { get; set; }

    public string? Email { get; set; }

    public string? PhoneNumber { get; set; }

    public string? Password { get; set; }

    public ulong IsDeleted { get; set; }

    public virtual ICollection<Trainerschedule> Trainerschedules { get; set; } = new List<Trainerschedule>();
}

File: web-app/src/app/services/types/Trainer.ts
import { Trainerschedule } from "./Trainerschedule";

export type Trainer = {
  Id: number;
  Name?: string;
  Email?: string;
  PhoneNumber?: string;
  Password?: string;
  IsDeleted: boolean;
  TrainerType?: string;
  CompanyName?: string;
  Trainerschedules: Trainerschedule[];
};
